b2sdk/file_version.py

```diff
@@ -86,7 +86,13 @@ def __init__(
         self.replication_status = replication_status
 
         if SRC_LAST_MODIFIED_MILLIS in self.file_info:
-            self.mod_time_millis = int(self.file_info[SRC_LAST_MODIFIED_MILLIS])
+            try:
+                if SRC_LAST_MODIFIED_MILLIS in self.file_info:
+                    self.mod_time_millis = int(self.file_info[SRC_LAST_MODIFIED_MILLIS])
+            except ValueError as e:
+                raise ValueError(
+                    'Could not convert %s to int. %s' % (self.file_info[SRC_LAST_MODIFIED_MILLIS], e)
+                )
```

Comment:
This makes the program stop in some cases where we'd want it to keep running, such as if an incopatible client uploads a file with a non-int `src_last_modified_millis`. The affected bucket could then not be used by sdk/cli.

An alternative solution to this problem would be to, in such case:
- emit a warning
- try to trim the number to an integer, maybe that works
- failing to trim, just leave the field unset so that the sdk and cli can still use the bucket

Reply:
Thank you for the valuable feedback. A modification has been made to give a warning instead of throwing an exception. When uploading to a bucket, the behavior shows that the upload is successful then the exception is thrown. However, the alternative presented to emit a warning ensures that the desired result is maintained. 

Let me know if there's a different thought here, but a warning would be more useful since the current environment throws an exception, as shown in issue #368.

Reply:
This modification has been reverted and the program is now crashing again with a ValueError instead of emiting a warning.